### ISSUE TITLE
Clipboard optimization fails

### ISSUE THREAD

**Issue by mehapps** [FuzzyIdeas/Clop#33]
Hey there, sorry for the bad screenshot - for some reason I couldn't screenshot the error so it's a picture from my phone. I've been using clop and I try to take a screenshot but it just shows optimization failed and shows this error, ran from Terminal as well but don't see any errors. Any thoughts? Is this a known issue? I'm on Sonoma 14.6.1 on a MacBook Pro 16-inch, 2019. Thanks!

![IMG_D2835D1213FE-1](https://github.com/user-attachments/assets/b3e9d859-d8f9-490d-be63-83f78362ac61)

**Comment by alin23** [comment]
Try collecting logs then do the action that causes this error to appear:

```
log stream --level debug --source --style compact --predicate 'subsystem BEGINSWITH "com.lowtechguys.Clop"' | ~/Desktop/clop.log
```

**Comment by mehapps** [comment]
Looks like it works now! I will report back if anything happens.

**Comment by mehapps** [comment]
Just an update, doesn't work anymore - something seems to be consistently overwriting it? Thinking about writing a Python helper to just fix that lol

**Comment by alin23** [comment]
The real fix would be to find what other app or process is deleting that Cache folder because Clop has no such code that could do that. 

I don't have time these days to add the folder check and recreate in Clop, but I hope to do this as soon as possible. 

Maybe the Crescendo app or [FSMonitor](https://fsmonitor.com) with a filter on that specific Clop folder would shed some light on what deletes Cache folders. I would be very curious myself to find out.

**Comment by mehapps** [comment]
Sounds good, I wasn't able to figure out why so for the meantime  I went ahead and created a helper app which seems to work good for me
https://github.com/mehapps/Clop-helper/releases/tag/v1.0.0

**Comment by reslear** [comment]
in terminal write: 

```bash
defaults write com.lowtechguys.Clop allowClopToAppearInScreenshots -bool true
```

and restart app, also @alin23  maybe add to instructions?

**Comment by alin23** [comment]
Fixed in [v2.6.1](https://github.com/FuzzyIdeas/Clop/releases/tag/v2.6.1)

@reslear I don't have any instructions page/place for technical and obscure stuff like this. I only use that setting when doing demos. I guess I could make the setting public somewhere in the Settings window if it's needed that often.

**Comment by mehapps** [comment]
FYI - is it possible that the issue is caused by the periodic cleanup setting within Clop? I turned it off and haven't seen the issue since
